src/components/App/index.js

```diff
@@ -83,13 +86,38 @@ class ModalApp extends Component {
         this.props.actions.setIdDocumentType(enabledDocs[0])
       }
     }
+    if (!options.mobileFlow && token && token !== prevToken) {
+      const jwtUrls = fetchUrlsFromJWT(token)
```

Comment:
[Minor] Odd function naming `fetchUrlsFromJWT` - initially I thought that should be an async call to some BE so here async/await would be needed, but then looked up at the function... IMO it should be `getUrlsFromJWT` or something similar.